avail/src/main/kotlin/avail/descriptor/atoms/AtomDescriptor.kt

```diff
@@ -486,7 +486,12 @@ open class AtomDescriptor protected constructor (
 		 * they were, we still wouldn't want to serialize one launched from a
 		 * debugger, so this atom itself doesn't need to be serializable.
 		 */
-		DONT_DEBUG_KEY("don't debug", heritable = true);
+		DONT_DEBUG_KEY("don't debug", heritable = true),
+
+		/**
+		 * The atom used as a property key under which to store a [FileHandle].
+		 */
+		RESOURCE_KEY("resource key"),;
```

Comment:
Crap, this leaked in on accident. I was trying to tie off the sample projects before moving forward with the Resource work. I though I avoided checking that in...